ts/src/woo.ts

```diff
@@ -2221,8 +2299,8 @@ export default class woo extends Exchange {
             'datetime': this.iso8601 (timestamp),
             'currency': code,
             'amount': this.safeNumber (transfer, 'amount'),
-            'fromAccount': fromAccount,
-            'toAccount': toAccount,
+            'fromAccount': this.safeString (transfer, 'from_application_id'),
```

Comment:
should we use the raw values in the response?

Reply:
do you mean from account and to account?

Reply:
usually we unify that to "main/spot/etc"